app/vlstorage/main.go

```diff
@@ -141,9 +144,19 @@ func initNetworkStorage() {
 
 	logger.Infof("starting insert service for nodes %s", *storageNodeAddrs)
 	netstorageInsert = netinsert.NewStorage(*storageNodeAddrs, authCfgs, isTLSs, *insertConcurrency, *insertDisableCompression)
+	netstorageInsertMetrics = metrics.NewSet()
+	netstorageInsertMetrics.RegisterMetricsWriter(func(w io.Writer) {
+		netstorageInsert.WriteMetrics(w)
+	})
+	metrics.RegisterSet(netstorageInsertMetrics)
```

Comment:
The `netstorageInsertMetrics` and `netstorageSelectMetrics` metrics sets must be de-registered in the `Stop()` function with the `metrics.UnregisterSet()`. See how it is made for the `localStorageMetrics` metrics set. This is needed in order to be able to call `Init()` and `Stop()` multiple times. It would be great adding a test for `vlstorage.Init()` and `vlstorage.Stop()` calls in a loop this inside `app/victoria-logs` in a separate pull request.